gpu-aware-scheduling/cmd/gas-scheduler-extender/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/intel/telemetry-aware-scheduling/extender"
+	"github.com/intel/telemetry-aware-scheduling/gpu-aware-scheduling/pkg/gpuscheduler"
+	"k8s.io/klog/v2"
+)
+
+func main() {
+	var (
+		kubeConfig, port, certFile, keyFile, caFile string
+		unsafe                                      bool
+	)
+
+	flag.StringVar(&kubeConfig, "kubeConfig", "/root/.kube/config", "location of kubernetes config file")
```

Comment:
We might think of doing some common flag handling, though this isn't exactly overkill. The flags that are used by the extender - most of the flags here really - should probably go in the extender package.

Reply:
sounds good to me